core/controlplane/config/templates/cloud-config-controller

```diff
@@ -9,6 +9,20 @@ coreos:
     etcd_keyfile: /etc/kubernetes/ssl/etcd-client-key.pem
 
   units:
+{{- range $thisunit := .Controller.CustomUnits}}
```

Comment:
I prefer `CustomSystemdUnits` rather than `CustomUnits` - Just saying it "unit" makes things a bit ambiguous between this and ones in spot fleets.